File: test-schemas-domain-classes/src/main/scala/testdomains/gratefuldead/PropertyNames.java
package testdomains.gratefuldead;

import java.util.HashSet;
import java.util.Set;

public class PropertyNames {


public static final String NAME = "name";


public static final String PERFORMANCES = "performances";


public static final String SONGTYPE = "songType";


public static final String WEIGHT = "weight";

public static Set<String> ALL = new HashSet<String>() {{
add(NAME);
add(PERFORMANCES);
add(SONGTYPE);
add(WEIGHT);
}};

}

File: test-schemas-domain-classes/src/main/scala/testdomains/gratefuldead/PropertyNames.scala
package testdomains.gratefuldead

import java.util.{HashSet, Set}
import scala.jdk.CollectionConverters.SeqHasAsJava

object PropertyNames {

  /** This is a property comment */
  val Name: String         = "name"
  val Performances: String = "performances"
  val Songtype: String     = "songType"
  val Weight: String       = "weight"

  val All: Set[String] = new HashSet[String](Seq(Name, Performances, Songtype, Weight).asJava)
}
